scripts/update-requirements

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+
+# To update the requirements files with sha256sums from our local PyPI.
+
+import os
+import sys
+import glob
+import hashlib
+from pprint import pprint
+
+
+def main():
+    PKG_DIR = os.environ["PKG_DIR"]
+    req_path = os.path.join(PKG_DIR, "requirements.txt")
+    reqdev_path = os.path.join(PKG_DIR, "requirements-dev.txt")
+    for path in [req_path, reqdev_path]:
+        if not os.path.exists(path):
+            print("Missing {0}.".format(path))
+            sys.exit(1)
+
+    # First remove index line and any PyQt or sip dependency
+    cleanup(req_path)
+    cleanup(reqdev_path)
+
+    # Now let us update the files along with the sha256sums from localwheels
+    add_sha256sums(req_path)
+    add_sha256sums(reqdev_path)
+
+
+def add_sha256sums(path: str) -> None:
+    """Adds all the required sha256sums to the wheels
+    
+    :param path: path to the requirements file.
+    :type path: str
+    :return: None
+    :rtype: None
+    """
+    files = []
+
+    with open("sha256sums.txt") as fobj:
+        lines = fobj.readlines()
+
+    for line in lines:
+        line = line.strip()
+        # Skip any sources
+        if line.find(".whl") == -1:
+            continue
+        files.append(line.split())
+
+    with open(path) as fobj:
+        lines = fobj.readlines()
+
+    newlines = []
+    missing_wheels = []
+    for line in lines:
+        line = line.strip()
+        words = line.split()
+        values = words[0].split("==")
+        values[1] = values[1].strip(';')
+        packagename = "-".join(values)
+        othername = "-".join([values[0].replace("-", "_"), values[1]])
+
+        for name in files:
+            lowername = name[1].lower()
+            digest = name[0]
+            package_othername = packagename.replace("-", "_")
+            # Now check if a wheel is already available
+            if lowername.startswith(packagename) or lowername.startswith(
+                othername
+            ):
+                # Check if the sha256sum is already there or not
+                if digest not in line:
+                    # Now add the hash to the line
+                    line = "{} --hash=sha256:{}\n".format(line, digest)
+        newlines.append(line)
+        if line.find("--hash") == -1:  # Missing wheel
+            missing_wheels.append(values)
+
+    # Do not update the file if missing wheels
+    if missing_wheels:
+        print("The following dependent wheel(s) are missing:")
+        for value in missing_wheels:
+            print("{0}=={1}".format(value[0], value[1]))
+
+        print("\nPlease build the wheel by using the following command:\n")
+        print("\tPKG_DIR={0} make build-wheels\n".format(os.environ["PKG_DIR"]))
+        print("Then sync the newly built wheels and sources to the s3 bucket.")
+        print("Also update the index HTML files accordingly and sync to s3.")
```

Comment:
Let's link in the README this issue: https://github.com/freedomofpress/securedrop-debian-packaging-guide/issues/6 

otherwise this is buried and is difficult for people to find

Reply:
I've done this in the code now